target/linux/ramips/dts/mt7621_mikrotik_routerboard-760igs.dts

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621_mikrotik_routerboard-7xx.dtsi"
+
+/ {
+	compatible = "mikrotik,routerboard-760igs", "mediatek,mt7621-soc";
+	model = "MikroTik RouterBOARD 760iGS";
+
+	aliases {
+		led-boot = &led_pwr;
+		led-failsafe = &led_pwr;
+		led-running = &led_pwr;
+		led-upgrade = &led_pwr;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_pwr: pwr {
+			label = "routerboard-760igs:blue:pwr";
+			gpios = <&gpio 16 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		sfp {
+			label = "routerboard-760igs:blue:sfp";
+			gpios = <&gpio 9 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	sfp1: sfp1 {
+		compatible = "sff,sfp";
+		i2c-bus = <&i2c>;
+		los-gpios = <&gpio 6 GPIO_ACTIVE_HIGH>;
+		mod-def0-gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+		tx-disable-gpios = <&gpio 10 GPIO_ACTIVE_HIGH>;
+		maximum-power-milliwatt = <1000>;
+	};
+};
+
+&mdio {
+	ephy7: ethernet-phy@7 {
+		reg = <7>;
+		sfp = <&sfp1>;
+	};
+};
+
+&gmac1 {
+	status = "okay";
+	label = "sfp";
+	phy-handle = <&ephy7>;
+};
+
+&i2c {
+	status = "okay";
+};
+
+&state_default {
+	gpio {
+		/* gpio7 (uart3 group) goes high when
+		 * port5 (PoE out) is cabled to a
+		 * Mikrotik PoE-in capable port,
+		 * such as port1 on another rb760iGS */
+		groups = "uart2", "uart3", "jtag", "wdt";
+		function = "gpio";
+	};
+};
+
+&spi0 {
+	zt2046q@1 {
+		/* touchscreen controller used only for
+		 * voltage & temperature monitoring.
+		 * kmod-input-touchscreen-ads7846
+		 * interrupt unknown / unneeded,
+		 * picked a presumed-unused gpio.
+		 * pendown unknown / unneeded,
+		 * picked a presumed-unused gpio. */
+		reg = <1>;	/* CS1 */
+		/* compatible = "zt2046q", "ti,ads7846";
+		 * spi-max-frequency = <1000000>;
+		 * interrupt-parent = <&gpio>;
+		 * interrupts = <31 IRQ_TYPE_NONE>;
+		 * pendown-gpio = <&gpio 30 0>; */
```

Comment:
I don't really like this. If it's not supported anyway, consider just removing it here, and bury the information in the commit message and/or Wiki in a suitable way.

Reply:
Done, cut. I think I have enough about it in the commit message.